cosmos/providers/dbt/core/operators/local.py

```diff
@@ -111,6 +111,20 @@ def store_compiled_sql(
         ).delete()
         session.add(rtif)
 
+    def handle_profiles_yml(self, project_dir: str) -> None:
+        """
+        If there's a profiles.yml file in the project dir, remove it. Cosmos doesn't support
+        user-supplied profiles.yml files.
+        """
+        profiles_path = os.path.join(project_dir, "profiles.yml")
+        if os.path.exists(profiles_path):
+            logger.warning(
+                "Cosmos doesn't support user-supplied profiles.yml files. \
+                    Please use Airflow connections instead. \
+                    Ignoring profiles.yml…"
+            )
+            os.remove(profiles_path)
```

Comment:
@jlaneve, couldn't we ignore the `profiles_path` without deleting it..? I'm slightly worried about us removing a file which users wouldn't expect to be removed. Let's say they sometimes use DBT directly to troubleshoot things, etc. Or they are building the docker container from this folder so that they can use the K8s/Docker Cosmos `execution mode` (and still need `profiles.yml`).